docs/data/data-grid/components/JoyUISlots.tsx

```diff
@@ -0,0 +1,136 @@
+  colorSchemes: {
+    light: {
+      palette: {
+        primary: {
+          ...blue,
+          solidColor: 'var(--mui-palette-primary-contrastText)',
+          solidBg: 'var(--mui-palette-primary-main)',
+          solidHoverBg: 'var(--mui-palette-primary-dark)',
+          plainColor: 'var(--mui-palette-primary-main)',
+          plainHoverBg:
+            'rgba(var(--mui-palette-primary-mainChannel) / var(--mui-palette-action-hoverOpacity))',
+          plainActiveBg: 'rgba(var(--mui-palette-primary-mainChannel) / 0.3)',
+          outlinedBorder: 'rgba(var(--mui-palette-primary-mainChannel) / 0.5)',
+          outlinedColor: 'var(--mui-palette-primary-main)',
+          outlinedHoverBg:
+            'rgba(var(--mui-palette-primary-mainChannel) / var(--mui-palette-action-hoverOpacity))',
+          outlinedHoverBorder: 'var(--mui-palette-primary-main)',
+          outlinedActiveBg: 'rgba(var(--mui-palette-primary-mainChannel) / 0.3)',
+        },
+        neutral: {
+          ...grey,
+        },
+        divider: 'var(--mui-palette-divider)',
+        text: {
+          tertiary: 'rgba(0 0 0 / 0.56)',
+        },
+      },
+    },
+    dark: {
+      palette: {
+        primary: {
+          ...blue,
+          solidColor: 'var(--mui-palette-primary-contrastText)',
+          solidBg: 'var(--mui-palette-primary-main)',
+          solidHoverBg: 'var(--mui-palette-primary-dark)',
+          plainColor: 'var(--mui-palette-primary-main)',
+          plainHoverBg:
+            'rgba(var(--mui-palette-primary-mainChannel) / var(--mui-palette-action-hoverOpacity))',
+          plainActiveBg: 'rgba(var(--mui-palette-primary-mainChannel) / 0.3)',
+          outlinedBorder: 'rgba(var(--mui-palette-primary-mainChannel) / 0.5)',
+          outlinedColor: 'var(--mui-palette-primary-main)',
+          outlinedHoverBg:
+            'rgba(var(--mui-palette-primary-mainChannel) / var(--mui-palette-action-hoverOpacity))',
+          outlinedHoverBorder: 'var(--mui-palette-primary-main)',
+          outlinedActiveBg: 'rgba(var(--mui-palette-primary-mainChannel) / 0.3)',
+        },
+        neutral: {
+          ...grey,
+        },
+        divider: 'var(--mui-palette-divider)',
+        text: {
+          tertiary: 'rgba(255 255 255 / 0.5)',
+        },
+      },
+    },
+  },
+  fontFamily: {
+    display: '"Roboto","Helvetica","Arial",sans-serif',
+    body: '"Roboto","Helvetica","Arial",sans-serif',
+  },
+  shadow: {
+    xs: `var(--mui-shadowRing), ${muiTheme.shadows[1]}`,
+    sm: `var(--mui-shadowRing), ${muiTheme.shadows[2]}`,
+    md: `var(--mui-shadowRing), ${muiTheme.shadows[4]}`,
+    lg: `var(--mui-shadowRing), ${muiTheme.shadows[8]}`,
+    xl: `var(--mui-shadowRing), ${muiTheme.shadows[12]}`,
+  },
+});
+
+const mergedTheme = deepmerge(joyTheme, muiTheme) as unknown as ReturnType<
+  typeof extendMuiTheme
+>;
+
+mergedTheme.unstable_sxConfig = {
+  ...muiSxConfig,
+  ...joySxConfig,
+};
+
+export default function JoyUISlots() {
```

Comment:
There can be Joy UI with other more complex components, to prefix?

```suggestion
export default function GridJoyUISlots() {
```